app/controllers/membership_applications_controller.rb

```diff
@@ -68,6 +68,11 @@ def information
 
   end
 
+  def destroy
+    @membership_application.destroy
+    redirect_to membership_applications_url, notice: t('membership_applications.application_deleted')
```

Comment:
Yes, but it was the swedish one you said something about, is that still correct?
Answer is no:
`application_deleted: Ansökan raderat.`
It was correct, thus should be
`application_deleted: Ansökan raderad.`